tekton/workshop/content/lab08.md

```diff
@@ -1,32 +1,18 @@
 # Introduction  
 
-In this lab, we will add the `Unit Test` stage of the DevSecOps pipeline
+In this lab, we will add the **Unit Test** stage of the DevSecOps pipeline
 
 ![Unit Test Stage](images/openshift-pipeline-unittest.png)
 
 # Add New Task to the Pipeline 
 
 Since the tests in a maven project are run directly by Maven, all we need is to add a new task to our pipeline that will call the `test` goal. We can reuse our existing `simple-maven` task and pass a parameter for the `GOAL` param in the task.
 
-The only thing that is different about the `test-app` task in the pipeline is that we are using the `runAfter` attribute so that the `test-app` task runs in *after* `build-app` instead of in parallel (this will come in handy very shortly).  > **NOTE** : since we're updating the existing pipeline YAML, pay attention to the indentation of the tasks - the new `test-app` step needs to be at the same indentation level as the `build-app` step.
-
+The only thing that is different about the `test-app` task in the pipeline is that we are using the `runAfter` attribute so that the `test-app` task runs in *after* `build-app` instead of in parallel (this will come in handy very shortly). Since we need to add an additional  step to the pipeline, we're going to **patch** our `Pipeline` object. We've already wired up a command string for you so you don't have to worry about managing indentation:
 
 ```yaml
```

Comment:
should change this to execute, too.

I see the "jankiness" workaround. Still like it :)